frontend/src/libs/components/form/form.tsx

```diff
@@ -39,7 +40,19 @@ const Form = <T extends FieldValues = FieldValues>({
 
   const createInputs = (): JSX.Element[] => {
     return fields.map((field, index) => (
-      <Input {...field} control={control} errors={errors} key={index} />
+      <div key={index}>
+        {field.type === 'dropdown' && field.options ? (
+          <Dropdown
+            options={field.options}
+            name={field.name}
+            control={control}
+            errors={errors}
+            label={field.label}
+          />
+        ) : (
+          <Input {...field} control={control} errors={errors} />
+        )}
+      </div>
```

Comment:
looks like it should be switch

Reply:
function renderField<T extends FieldValues = FieldValues>(
  field: FormField<T>,
  control: Control<T, null>,
  errors: FieldErrors<T>,
): JSX.Element {
  switch (field.type) {
    case 'dropdown': {
      const { options, name, label } = field;

      return (
        <Dropdown
          options={options ?? []}
          name={name}
          control={control}
          errors={errors}
          label={label}
        />
      );
    }
    case 'number':
    case 'text':
    case 'email':
    case 'password': {
      return <Input {...field} control={control} errors={errors} />;
    }
    default: {
      return <Input {...field} control={control} errors={errors} />;
    }
  }
} like this?